src/App.jsx

```diff
@@ -1,104 +1,26 @@
 import './App.scss';
```

Comment:
The task requires that styles from `App.scss` should be moved to separate files, such as `PostList.scss`. It seems the `App.scss` file still exists and contains styles, indicating that the styles have not been fully separated as required.